config/testdata/conf.good.yml

```diff
@@ -109,7 +109,8 @@ scrape_configs:
 - job_name: service-kubernetes
 
   kubernetes_sd_configs:
-  - api_servers:
+  - kind: service_endpoints
```

Comment:
From our last discussion you were very much focused on the job label being set correctly.
Doing pod discovery needs a lot of customization to generate a job label and we won't be able to provide something generally meaningful here.

I don't see an issue with service endpoints tbh – it's exactly what we discussed (wanting to monitor services, except for special use cases) minus the pod-specific labels.
Can you explain what's sub-optimal here?